README.md

```diff
@@ -10,27 +10,9 @@ And because we have absolutely no idea how to write a programming language, we'r
 # Prerequisites
 
 - You need composer to download the library
-- Your REST API has to return JSON and to adhere to the following rules:
-    - Use POST to create new data
-        - Urls have the following format: ```http://www.host.de/path/to/api/entityName```
-        - Use the request body to receive data
-        - Respond with HTTP code 200 if successful
-        - Fill the response body with the given payload plus an id
-    - Use PUT to change data
-        - Urls have the following format: ```http://www.host.de/path/to/api/entityName/<id>```
-        - Use the request body to receive data
-        - Respond with HTTP code 200 if successful
-        - Fill the response body with the given payload
-    - Use DELETE to remove data
-        - Urls have the following format: ```http://www.host.de/path/to/api/entityName/<id>```
-        - The request body must be empty
-        - Respond with HTTP code 204 if successful
-        - The response body must be empty
-    - Use GET to receive data
-        - Urls have the following format: ```http://www.host.de/path/to/api/entityName/<id>``` or ```http://www.host.de/path/to/api/entityName```
-        - Use HTTP query strings as filters
-        - Respond with HTTP code 200 if successful
-        - Fill the response body with the read data
+- Your REST API has to return JSON
+- GET, PUT and POST requests must be answered with 200 if successful
+- DELETE requests must be responded with 204 if successful
```

Comment:
must be responded to oder a response for a DELETE request needs to contain a status code of 204 if successful